index.html

```diff
@@ -165,6 +165,30 @@ <h4 class="connect">LET'S CONNECT</h4>
 </ul>
 </Section>
 
-  
+<div class="container1">
+  <h2 class="contact-us-heading">Contact me</h2>
+  <p>If you have an application you are interested
+    in developing, a feature that you need built or a project that needs coding.
+    I’d love to help with it!</p>
+<div class="form">
+  <form action="#" method="post">
```

Comment:
- [ ] It is required to integrate your for with formspree service where the `action` atttibute in your form would point to. You must create an account on the [formspree](https://formspree.io/html/) site where you would be given an id that will be used in your form. the action attribute on your form will look like this :point_down: 

```
  <form action="https://formspree.io/f/{form_id}" method="post">
```
Kindly work on that.